popup/popup.js

```diff
@@ -590,7 +599,8 @@ Object.assign(handleEvent, {
   },
 
   copyContent(event) {
-    const target = event.target;
+    event.preventDefault();
+    const target = document.activeElement;
```

Comment:
Why `event.target` is changed to `document.activeElement`? It might be even better to change it to `event.target.closest('.copy')`.

Reply:
Again, the changes here already exist in the master. IIRC, I used `document.activeElement` to facilitate some of the keyboard navigation. Possibly the focus trap? What's the difference?